pmlc/rt/executable.cc

```diff
@@ -71,8 +78,7 @@ std::string makeCWrapperFunctionName(StringRef name) {
 // Define an interface function that wraps all the arguments of the original
 // function and all its results into an i8** pointer to provide a unified
 // invocation interface.
-static std::string packFunctionArguments(llvm::Module *module,
-                                         StringRef entry) {
+std::string packFunctionArguments(llvm::Module *module, StringRef entry) {
```

Comment:
Please follow MLIR guidelines. Functions are static and types go into anonymous namespace

Reply:
Done.